### PR TITLE
[EN] Updating Enum.uniq/1 to Enum.uniq_by/2

### PR DESCRIPTION
**Pull request by dyancarlos**
Updating depreciated method.

### CODE REVIEW

File: en/lessons/basics/enum.md [hunk 1]
```diff
@@ -184,11 +184,13 @@ iex> Enum.sort([%{:count => 4}, %{:count => 1}])
...
-We can use `uniq/1` to remove duplicates from our enumerables:
+We can use `uniq_by/2` to remove duplicates from our enumerables:
 
 ```elixir
-iex> Enum.uniq([1, 2, 3, 2, 1, 1, 1, 1, 1])
+iex> Enum.uniq_by([1, 2, 3, 2, 1, 1, 1, 1, 1], fn x -> x end)
 [1, 2, 3]
 ```
+
+This function was previously known as `uniq/1`, which is obsolete from Elixir 1.4, but is still available (with alerts).
```
Comment (doomspork) [inline en/lessons/basics/enum.md]: Let's just make the change from `uniq/1` to `uniq_by/2` and skip this note, that way we don't need to come back later and remove it 👍

**Comment by dyancarlos** [comment]
Thanks @doomspork, I'm really happy to be able to help!  I just made the changes :D

**Comment by doomspork** [comment]
Thank you @dyancarlos!  Sorry for the delay, I've been away on holiday 👍

**Comment by dyancarlos** [comment]
No problem @doomspork , thanks! :D

**Pull request merged**